test/unit-test/main.tf

```diff
@@ -59,6 +59,10 @@ module "dns_mod" {
 }
 
 resource "aws_alb" "testalb" {
+  # A number of tests to ignore to stop errors in the overnight check
+  #checkov:skip=CKV_AWS_150 "Ensure that Load Balancer has deletion protection enabled - Not applicable in the test"
```

Comment:
`# checkov:skip=CKV_AWS_150 - Delete protection not applicable in the test
`

Could you add a space after each hashtag / number symbol, and I dont think you need the double danglers at all, but up to you. Looks okay!